evap/results/exporters.py

```diff
@@ -105,26 +112,37 @@ def export(self, response, selection_list, include_not_enough_voters=False, incl
                 for questionnaire_result in collect_results(evaluation).questionnaire_results:
                     if all(not question_result.question.is_rating_question or question_result.counts is None for question_result in questionnaire_result.question_results):
                         continue
-                    results.setdefault(questionnaire_result.questionnaire.id, []).extend(questionnaire_result.question_results)
-                    used_questionnaires.add(questionnaire_result.questionnaire)
```

Comment:
if we export for a contributor then we want to include only general and personal results. so we add results if `not contributor` (a "normal" semester export), `questionnaire_result.contributor is None` (result for a general question) or `questionnaire_result.contributor == contributor` (the result is for the current contributor).
otherwise we skip this result completely.